tools/flaky_test_monitor/process_results.go

```diff
@@ -218,6 +231,20 @@ func postProcessTestRun(packageResultMap map[string]*PackageResult) {
 		}
 
 		for _, testResults := range packageResult.TestMap {
+			for j, testResult := range testResults {
+				// found nil test - test with no result - due to `go test` bug
+				// see https://www.notion.so/dapperlabs/Tests-With-No-Result-2caa9ce6f69c4364a4df53f70ccb9255
+				if testResult.Result == "" {
+					// separate nil test results from regular test results
+					packageResult.NilTests = append(packageResult.NilTests, testResult)
+					// reslice test results without nil test - need to be careful that testResults slice isn't too small to be resliced
+					if len(testResults) >= j+1 {
```

Comment:
interesting, i always preffered `len(testResults) > j`, but they accomplish the same thing.